feature/#3_archivo_prueba2.txt

faoalvarado5 opened the pull request:
Se añadió un segundo ejemplo de prueba llamado"prueba2.txt"

faoalvarado5 committed:
feature/#3_archivo_prueba2.txt

Se añadió un segundo ejemplo de prueba llamado"prueba2.txt"

natefedez commented on src/resources/prueba2.txt:
```diff
@@ -0,0 +1,27 @@
+clase CribaEratostenes {
```
El archivo de prueba se ve bien, faltaría el resultado de ejecución de la prueba2.txt

faoalvarado5 committed:
Correccion

Se agrego un archivo con el resultado de la ejecucion de la prueba agregada anteriormente.

natefedez commented on src/resources/resultado_ejecucion_prueba2.txt:
```diff
@@ -0,0 +1,209 @@
+C:\Users\Fabrizio\AppData\Local\Programs\Python\Python38-32\python.exe C:/Users/Fabrizio/Documents/GitHub/PP1_Analizador_Lexico_IC5701/src/main/Main.py prueba2.txt
```
Todo bien, todo correcto y...
